python/GafferUI/GLWidget.py

```diff
@@ -464,6 +464,10 @@ def shape( self ) :
 			self.__shape = QtGui.QPainterPath()
 			if self.widget() :
 				pixmap = self.widget().grab()
+				if pixmap.size() != widget.size() :
+					# Account for the widget being grabbed at a higher resolution
+					# when using high resolution displays.
+					pixmap = pixmap.scaled( pixmap.size() / 2 )
```

Comment:
Why do we have to hardcode to half size, rather than using widget.size()?  There's probably a good reason, but at first glance, it seems like if we can use widget.size() to detect if the pixmap.size() isn't right, we should be able to use it to figure out the right size?

Reply:
And why did I reference the non-existent `widget` variable instead of `self.widget()`? Sorry, in a rush last night I pushed some weird hybrid from my original testing hack and the final version. Should be OK now...